src/library/templates/library/tab_library_quests.html

```
@@ -4,54 +4,90 @@
+        </td>
+        <td class="col-sm-1 col-xs-2 text-center">
+          {{ q.xp }}{% if q.xp_can_be_entered_by_students %}+{% endif %}
+        </td>
+        <td class="col-sm-2 text-center hidden-xs">
+          <small>{% if q.campaign %}{{ q.campaign }}{% endif %}</small>
+        </td>
+        <td class="col-sm-2 text-center hidden-xs">
+          <small>{{ q.tags.all|join:", "|default:"-" }}</small>
+        </td>
+        <td class="col-sm-1 hidden-xs text-muted" data-field="status_icons">
+            {% if q.max_repeats != 0 %}<i title='Repeat: this quest is repeatable' class='icon-spacing fa fa-fw fa-undo'></i>
```

HTMLHint (1.5.0)

[error] The value of attribute [ title ] must be in double quotes.

(attr-value-double-quotes)